composer/admin.py

```diff
@@ -28,9 +33,127 @@ def __init__(self, *args, **kwargs):
         except (AttributeError, KeyError):
             styles = []
         styles.append(("tile", "Tile"))
+        styles = self.get_existing_styles(styles)
         styles.sort()
         self.fields["style"].widget = forms.widgets.Select(choices=styles)
 
+    def get_existing_styles(self, current_styles):
+        # Return the current styles if the composer setting or the specific key
```

Comment:
Must be a docstring. Also, it doesn't describe all the things it can return. What happens if the key is set in settings? What is returned?